src/framework/uicomponents/qml/MuseScore/UiComponents/internal/GridViewSection.qml

```diff
@@ -1,20 +1,20 @@
-import QtQuick 2.12
+import QtQuick 2.15
 
 Item {
-    property Component sectionDelegate: Item {}
+    id: root
+    property alias sectionDelegate: loader.sourceComponent
 
     Loader {
+        id: loader
         anchors.fill: parent
 
         property var itemModel: null
         property int itemIndex: 0
-        sourceComponent: sectionDelegate
 
-        onStatusChanged: {
-            if (status === Loader.Ready) {
-                itemModel = Qt.binding( function() { return Boolean(modelData) ? modelData : null });
-                itemIndex = index
-            }
+        onLoaded: {
+            itemModel = Qt.binding( function() { return Boolean(modelData) ? modelData : null });
+            itemIndex = index
+            root.visible = Qt.binding( function() { return Boolean(item) ? item.visible : false })
```

Comment:
@cbjeukendrup 
Can you explain why you added this code?

Sections are not displayed on the master because of this code

Reply:
Without this code, a GridViewSection was visible even when it's content (the separator in the NoteInputBar) is not visible, which causes that an empty space is shown, and an extra `spacing` is added by the Column or Row containing the GridViewSection (see GridViewSectional.qml, line 65).

Also, I'm not sure whether this code is the cause that sections are not shown, because in the sample grid view on the DevTools page, they are shown correctly. 
I think the cause is that the `state` in NoteInputBar.qml is not correctly set initially; because when you trigger a state change by dragging the toolbar to a location where it is vertical, the separators are shown correctly. I also believe they are still shown correctly with the old docking system, but not in the new one. So I think initially a signal about `orientationChanged` is somehow not being emitted.